klarna-on-demand/src/main/java/com/klarna/ondemand/RegistrationActivity.java

```diff
@@ -66,4 +105,47 @@ public void onClick(DialogInterface dialog, int id) {
     protected int homeButtonResultCode() {
         return RESULT_CANCELED;
     }
+
+    //region LoaderManager.LoaderCallbacks<Cursor> implementation
+
+    @Override
+    public Loader<Cursor> onCreateLoader(int id, Bundle arguments) {
```

Comment:
All public methods must be documented. Either de-publicize them, or add some docs.